app/tests/utils/mockFormData.ts

```diff
@@ -211,6 +213,33 @@ const mockFormData = {
     contractorPersonMonthsToBeCreated: 1,
     hoursOfContractorEmploymentPerWeek: 1,
   },
+  acknowledgements: {
+    acknowledgementsList: [
+      acknowledgementsEnum[0],
+      acknowledgementsEnum[1],
+      acknowledgementsEnum[2],
+      acknowledgementsEnum[3],
+      acknowledgementsEnum[4],
+      acknowledgementsEnum[5],
+      acknowledgementsEnum[6],
+      acknowledgementsEnum[7],
+      acknowledgementsEnum[8],
+      acknowledgementsEnum[9],
+      acknowledgementsEnum[10],
+      acknowledgementsEnum[11],
+      acknowledgementsEnum[12],
+      acknowledgementsEnum[13],
+      acknowledgementsEnum[14],
+      acknowledgementsEnum[15],
+      acknowledgementsEnum[16],
+    ],
```

Comment:
```suggestion
    acknowledgementsList: acknowledgementsEnum,
```
why not simply using the whole array?